companies/bvg.json

```
@@ -0,0 +1,30 @@
+{
+    "slug": "bvg",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "public body",
+        "travel"
+    ],
+    "name": "Berliner Verkehrsbetriebe (BVG)",
+    "runs": [
+        "URBANIS GmbH",
+        "BT Berlin Transport GmbH (BT)",
+        "IOB Internationale Omnibusbahnhof-Betreibergesellschaft mbH",
+        "BVG Projekt GmbH",
+        "BVG Beteiligungsholding Verwaltungsgesellschaft mbH",
+        "BVG Beteiligungsholding GmbH & Co. KG"
+    ],
+    "address": "Berliner Verkehrsbetriebe (BVG) - AöR -\nDatenschutzbeauftragter\nHolzmarktstraße 15-17\n10179 Berlin\nDeutschland",
```

```suggestion
    "address": "Holzmarktstraße 15-17\n10179 Berlin\nDeutschland",
```
Our generator adds the company line and the DSB, so we do not need it in the address.